lib/ex_ami/client.ex

```diff
@@ -367,17 +362,16 @@ defmodule ExAmi.Client do
   defp connecting_timer(cnt) when cnt < 20, do: 30_000
   defp connecting_timer(_), do: 60_000
 
-  defp validate_salutation("Asterisk Call Manager/1.1\r\n"), do: :ok
-  defp validate_salutation("Asterisk Call Manager/1.0\r\n"), do: :ok
-  defp validate_salutation("Asterisk Call Manager/1.2\r\n"), do: :ok
-  defp validate_salutation("Asterisk Call Manager/1.3\r\n"), do: :ok
+  defp validate_salutation("Asterisk Call Manager/7.0." <> patch = saluation) do
```

Comment:
You need to spell salutation correctly throughout.

Reply:
Fixed.